kDriveCore/Data/Cache/DriveFileManager/DriveFileManagerConstants.swift

```diff
@@ -178,6 +178,7 @@ public class DriveFileManagerConstants {
                       UploadingSessionTask.self]
     )
 
+    // TODO: Remove me, as should remove the possibility to access realm directly
     /// realm db used for file upload
     public var uploadsRealm: Realm {
```

Comment:
This is not called a getRealm, but is does exactly that. Given the name differed, it was not part of the first refactor.

Reply:
cleaned